client-src/elements/chromedash-stack-rank.js

```diff
@@ -219,7 +237,8 @@ class ChromedashStackRank extends LitElement {
   renderStackRank(displayedList) {
     return html`
       ${displayedList.map((item) => html`
-        <li class="stack-rank-item" id="${item.property_name}">
+        <li class="stack-rank-item${(this.shouldHideObsolete && item.obsolete) ? ' stack-rank-item-hidden' : ''}"
```

Comment:
I think it would be OK for the class attribute to have a trailing space when the item is not hidden.  That would allow you to write the code more cleanly with a space before the `$` and no leading space inside the string constant `'stack-rank-item-hidden'`.